web/src/layouts/AppLayout.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link, Outlet } from 'react-router-dom'
+import { useState } from 'react'
+import { apiGet } from '../lib/api'
+import AccountModal from '../components/AccountModal'
+
+export default function AppLayout() {
+  const [modalOpen, setModalOpen] = useState(false)
+  const [loading, setLoading] = useState(false)
+  const [error, setError] = useState<string | null>(null)
+  const [user, setUser] = useState<User | null>(null)
+  
+  async function loadUser() {
+    try {
+      setError(null)
+      setLoading(true)
+      const me = await apiGet<User>('/api/v1/account/me')
```

Comment:
Every time the “Account” button is clicked, you’re calling /api/v1/account/me. The result isn’t stored anywhere beyond state, so you’re refetching the same info repeatedly.
Try React Query, or memoize it until logout.